pkg/tarutil/tar.go

```diff
@@ -260,23 +257,12 @@ func createFileInRoot(hdr *tar.Header, r io.Reader, rootDir string) error {
 // the file is omitted.
 type Filter func(hdr *tar.Header) bool
 
-// NoFilter does not filter or modify any files.
-func NoFilter(hdr *tar.Header) bool {
-	return true
-}
-
 // VerboseFilter prints the name of every file.
 func VerboseFilter(hdr *tar.Header) bool {
 	fmt.Println(hdr.Name)
 	return true
 }
 
-// VerboseLogFilter logs the name of every file.
```

Comment:
why remove this one? It seems useful.

Reply:
Because VerboseFilter and VerboseLogFilter doing the same thing, because VerboseFilter is used in tar command I decide to remove VerboseLogFilter.

```go
// VerboseFilter prints the name of every file.
func VerboseFilter(hdr *tar.Header) bool {
	fmt.Println(hdr.Name)
	return true
}

// VerboseLogFilter logs the name of every file.
func VerboseLogFilter(hdr *tar.Header) bool {
	log.Println(hdr.Name)
	return true
}
```

Reply:
They do the same thing, true. But Log is to stderr, and the other one is to stdout. Are you sure it's ok to remove one? I'll approve anyway. Let's hope nobody was using it.

Reply:
Thx, I missed that :) I will revert and add a test to log part.